### PR TITLE
Task 1308-1313 Amazon review 

### PR DESCRIPTION
**Pull request by cosmicishan**
task1308_amazonreview_category_correction_classification -> You're given a review from Amazon and category of the product based on the review, classify whether review matches the category. 
task1309_amazonreview_summary_correction_classification -> You're given a review from Amazon and a summary of the review, classify whether the summary matches the original review.
task1310_amazonreview_rating_classification -> You're given a review from Amazon. Your task is to generate a rating for the product on a scale of 1-5 based on the review.
task1311_amazonreview_rating_correction_classification -> You're given a review from Amazon and rating of the review, classify whether rating matches the review.
task1312_amazonreview_polarity_classification -> You are given a review of Amazon's food products. Your task is to divide them into two classes: negative or positive.
task1313_amazonreview_polarity_correction_classification.json -> You're given a review from Amazon and polarity of the review, classify whether review matches the polarity.

Please Review.

### CODE REVIEW

**Comment by aarunku5** [comment]
Thanks, similar comment across the tasks- the content is fine, could you split the definition and explanation sentences into shorter ones? Couple of grammar errors here and there as well in the explanations, so just double check that as well.

**Comment by cosmicishan** [comment]
Screenshot from 2021-10-13 14-55-24


@swarooprm I am getting this error message, can you help me with this? What do I need to add or remove? I am having this error in every single PR even the new one I added. I closed the previous one thinking it will probably be something wrong in that particular PR but seems like I am having conflicts in all PR. I don't understand the issue. I also show there was something wrong in PR sharing history with another PR which automatically adds some redundant tasks. Do you know the solution?

**Comment by Palipoor** [comment]
Hi,
You need to add these three to all your tasks:
```
    "Input_language": [
        "English"
    ],
    "Output_language": [
        "English"
    ],
    "Instruction_language": [
        "English"
    ]
```
If they're not in English, update the added fields accordingly.
Thanks

**Comment by danyaljj** [comment]
Please make sure to set the `"Categories": [""]` and `"Domains": [""]` fields according to [the taxonomy presented here](https://github.com/allenai/natural-instructions-expansion/blob/master/doc/task-hierarchy.md).

**Comment by cosmicishan** [comment]
> Hi, You need to add these three to all your tasks:
> 
> ```
>     "Input_language": [
>         "English"
>     ],
>     "Output_language": [
>         "English"
>     ],
>     "Instruction_language": [
>         "English"
>     ]
> ```
> 
> If they're not in English, update the added fields accordingly. Thanks

Thank you so much it worked.

**Comment by cosmicishan** [comment]
> Thanks, similar comment across the tasks- the content is fine, could you split the definition and explanation sentences into shorter ones? Couple of grammar errors here and there as well in the explanations, so just double check that as well.

I have fixed that now and double-checked all definitions and explanations. Can you review it again? Hope I didn't miss anything out.

File: tasks/task588_amazonfood_rating_classification.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "Categories": [
         "Classification"
     ],
-    "Definition": "In this task, you're given aa review from Amazon's food products. Your task is to generate a rating for the product on a scale of 1-5 based on the review. The rating means 1: extremely poor, 2: poor, 3: neutral, 4: good, 5: extremely good. ",
+    "Definition": "In this task, you're given a review from Amazon's food products. Your task is to generate a rating for the product on a scale of 1-5 based on the review. The rating means 1: extremely poor, 2: poor, 3: neutral, 4: good, 5: extremely good. ",
```
Comment (danyaljj) [inline tasks/task588_amazonfood_rating_classification.json:11]: 👍

File: tasks/README.md [hunk 2]
```diff
@@ -924,6 +924,12 @@ Name | Summary | Category | Domain | Input Language | Output Language
 `task1305_kpa_keypoint_matching_journalism_topic` | Given an argument and a keypoint under the topic "We should subsidize journalism", answer if the keypoint summarizes the argument | Classification
 `task1306_kpa_keypoint_matching_space_exploration_topic` | Given an argument and a keypoint under the topic "We should subsidize space exploration", answer if the keypoint summarizes the argument | Classification
 `task1307_kpa_keypoint_matching_vocational_education_topic` | Given an argument and a keypoint under the topic "We should subsidize vocational education", answer if the keypoint summarizes the argument | Classification
+`task1308_amazonreview_category_correction_classification` | You're given a review from Amazon and category of the product based on the review, classify whether review match the category. | Classification
```
Comment (danyaljj) [inline tasks/README.md]: what does `correction` imply in the task name?

Comment (danyaljj) [reply tasks/README.md]: (applies to most of the tasks here)

Comment (danyaljj) [reply tasks/README.md]: Also, for several tasks you have `"Text generation"` as their task category, even though they seem to be `Vericiation` or `Classification`.

**Comment by danyaljj** [comment]
Few suggestions, otherwise, it looks good.

**Comment by swarooprm** [comment]
@cosmicishan, can you address reviewer comments so that we can go ahead with this PR?

**Comment by cosmicishan** [comment]
All the changes are done.

Please review again. @danyaljj

**Comment by danyaljj** [comment]
Thanks!

**Pull request merged**